include/spirv/unified1/extinst.nonsemantic.clspvreflection.grammar.json

```diff
@@ -1,12 +1,14 @@
 {
-  "revision" : 4,
+  "revision" : 5,
   "instructions" : [
     {
       "opname" : "Kernel",
       "opcode" : 1,
       "operands" : [
         { "kind" : "IdRef", "name" : "Kernel" },
-        { "kind" : "IdRef", "name" : "Name" }
+        { "kind" : "IdRef", "name" : "Name" },
+        { "kind" : "IdRef", "name" : "NumArguments"},
+        { "kind" : "IdRef", "name" : "Flags" }
```

Comment:
Sorry there was a disconnect between the two reviews. These should both be optional for backwards compatibility. The validation for this instruction should eventually be updated to reject this instruction of the number of operands is not 2 in version 4 and 4 in version 5.

Reply:
I think the statement about the missing operands in the instruction set covers the optionality aspect in the spec language enough.